core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -81,7 +81,8 @@ class CityConstructions {
             stats.add(building.getStats(cityInfo.civInfo))
 
         for (unique in builtBuildingUniqueMap.getAllUniques()) when (unique.placeholderText) {
-            "[] Per [] Population in this city" -> stats.add(unique.stats.times(cityInfo.population.population / unique.params[1].toFloat()))
+            "[] Per [] Population []" -> if (cityInfo.matchesFilter(unique.params[2])) 
```

Comment:
lowercase - see CityStats:258

Reply:
Also, please keep the older version for backwards compatibility, but mark it with a "Deprecated at 3.15.17" comment

Reply:
I'm confused. In [this](https://github.com/yairm210/Unciv/pull/4084#issuecomment-859038672) comment in the old thread you seemed to imply that removing uniques wasn't a problem in general. So is deprecating to support backwards compatibility something that should be done in general, or just something for this specific case as this was a (probably) widely used unique?